lib/ansible/modules/cloud/azure/azure_rm_loadbalancer.py

```diff
@@ -632,58 +554,6 @@ def __init__(self):
                 elements='dict',
                 options=load_balancing_rule_spec
             ),
-            public_ip_address_name=dict(
```

Comment:
@felixfontein The verify fail was caused by deleting it, so why delete it? Thank you very much!

Reply:
Because the option was deprecated and was supposed to be removed in Ansible 2.9, according to the documentation and the `module.deprecate()` call. Please check whether that's correct.